index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,700;0,900;1,100;1,300&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="logo">
+        <img src="images/my logo.png" alt="">
+      </div>
+      <div class="menu-bar">
+        <img src="images/Enabledhamburger.png" alt="">
+      </div>
+    </nav>
+  
+   <section class="header">
+     <h2>I’m Mariane Glad to see you!</h2>
+     <main>
+       <article>
+            I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+       </article>
+
+       <div class="socials">
+         <h3>let's connect</h3>
+
+         <div class="links">
+           <a href="">
+             <img src="images/twitter.svg" alt="">
+           </a>
+           <a href="">
+             <img src="images/Linkedin icon.svg" alt="">
+           </a>
+           <a href="">
+             <img src="images/m.svg" alt="">
+           </a>
+           <a href="">
+             <img src="images/github.svg" alt="">
+           </a>
+            <a href="">
+              <img src="images/peace.svg" alt="">
+            </a>
+          </div>
```

Comment:
- [x] Please make sure that your social media links are displayed using list items. The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
  - for example
  ```html
  <ul>
   <li>
     <a><img src="./images/Linkedin icon.png" alt="none"></a>
   </li>
  </ul>
  ```
- [x] Please note that apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.